gui/src/components/LayoutComponents/SideBar.tsx

```diff
@@ -24,34 +35,55 @@ const SideBar: React.FC = () => {
     [pathname],
   );
 
+  const filteredOptions = useMemo(() => {
+    const projectType = getProjectTypeFromFramework(projectFramework);
+    if (projectType === projectTypes.DESIGN) {
+      return sidebarOptions.filter((option) =>
+        ['design', 'workbench'].includes(option.id),
+      );
+    } else if (projectType === projectTypes.BACKEND) {
+      return sidebarOptions.filter((option) =>
+        ['board', 'workbench', 'code', 'pull_request'].includes(option.id),
```

Comment:
not a good idea to have all the names here, create a constant  and below it an constant array with all the items and use it here ensuring in future more can be added without issue

Reply:
ok